Color/Colors.php

```diff
@@ -167,150 +167,162 @@ class Colors
      *
      * @return array
      */
-    public static function getAll()
+    public static function getAll($isInverted = true)
```

Comment:
> IMHO flipping the elements inside the method is also a BC break.

That's why I was suggesting doing 2 deprecation notices (see the 2 use cases). If we flip something, we must keep in mind that we will remove code in next major, and that when this code is removed, it would be great to have no flip at all left, don't you think?